algorithms/sorting/strand_sort.py

```diff
@@ -17,53 +17,46 @@
     Psuedo Code: https://en.wikipedia.org/wiki/Strand_sort
 """
 
-from collections import deque
 
-
-def sort(array):
-    if len(array) < 2:
-        return array
+def sort(unsorted):
+    if len(unsorted) < 2:
+        return unsorted
     result = []
-    while array:
-        sublist = [array.pop()]
+    while unsorted:
+        sublist = [unsorted.pop(0)]
+        leftovers = []
         last = sublist[0]
-        sub_append = sublist.append
-        leftovers = deque()
-        left_append = leftovers.append
-        for item in array:
+        sublist_append = sublist.append
+        leftovers_append = leftovers.append

```

Comment:
Thanks for the explanation, makes total sense. Can you just make a short comment above the assignment briefly explaining this? It only has to be a couple lines.